<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vehicle Power Measurement System</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>151</y>
     <width>1081</width>
     <height>561</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Voltage (Volt)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Current (Amps)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Power (Watt)</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>80</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Email/User name</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>80</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="genCSVButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>730</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate CSV</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Date (m/d/y):</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadDataButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QLabel" name="logLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>730</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Log: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deviceID">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Device ID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
